File: Quick_Sort.py
import Fill_List_Python

list = list()

Fill_List_Python.Fill_List(list)

print(list)

def Quick_Sort(list):
	length = len(list)
	if length <= 1:
		return list
	else:
		pivot = list.pop()
	lesser_list = []
	greater_list = []

	for i in list:
		if i > pivot:
				greater_list.append(i)
		else:
				lesser_list.append(i)
	return Quick_Sort(lesser_list) + [pivot] + Quick_Sort(greater_list)

print(Quick_Sort(list))

File: Quick_Sort.rb
require_relative "Fill_List_Ruby"

list = Array.new

Fill_List(list)
print(list)
def Quick_Sort(list)

return list if list.size <= 1

pivot = list.pop
lower_list = Array.new
higher_list = Array.new
list.each do |i|
	if i > pivot
		higher_list << (i)
	else
		lower_list << (i)
	end
end
return Quick_Sort(lower_list) + [pivot] + Quick_Sort(higher_list)
end
puts ""
print(Quick_Sort(list))